ui/settings-dialog.ui

```diff
@@ -192,6 +192,41 @@
        </item>
       </layout>
      </widget>
+     <widget class="QWidget" name="mLanguageTab">
+      <attribute name="title">
+       <string>Language</string>
+      </attribute>
+      <layout class="QVBoxLayout" name="languageTabLayout">
+       <item>
+        <layout class="QHBoxLayout" name="languageLayout">
+         <item>
+          <widget class="QLabel" name="mLanguageLabel">
+           <property name="text">
+            <string>Language (need restart)</string>
```

Comment:
It is hard to implement because it requires to add event filters for all the widgets.